File: 백준/Bronze/3062. 수 뒤집기/수 뒤집기.kt
fun main() {
    val n = readLine()!!.toInt()

    repeat(n) {
        val num = readLine()!!
        val reverseNum = num.reversed()

        val sum = num.toInt() + reverseNum.toInt()

        if (sum.toString() == sum.toString().reversed()) {
            println("YES")
        } else {
            println("NO")
        }
    }
}

File: 백준/Bronze/3062. 수 뒤집기/수 뒤집기.py
n = int(input())

for i in range(n):
    num = input()
    reverseNum = num[::-1]

    sum = int(num) + int(reverseNum)

    if str(sum) == (str(sum)[::-1]):
        print("YES")
    else:
        print("NO")
